Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:latest AS builder
+
+RUN go get github.com/vinyldns/vinyldns-cli \
```

Comment:
This is awesome! Thanks @marekfilip ! I think I'd prefer the `Dockerfile` use the `Makefile` tasks in building `vinyldns` such that the `Makefile` becomes the single source of truth for building `vinyldns`. How do you feel about that?

Reply:
Not a problem, will do! Image build time will be bit longer, thankfully it's Go so it's not a big deal.